### ISSUE TITLE
Some presets make Find bar button text hard to see

### ISSUE THREAD

**Issue by FayneAldan** [mozilla/FirefoxColor#592]
Some of the presets use toolbar transparency in such a way that the background image is primarily seen on the toolbar and shown behind text. However, when using such a method, it can cause the Find bar to have weird coloring, causing the button text to become invisible or hard to see. This can be seen in the two presets shown below:

![First Preset Toolbar](https://i.imgur.com/UsmFkDS.png)
![First Preset Find Bar](https://i.imgur.com/POId2tq.png)

![Second Preset Toolbar](https://i.imgur.com/D54jtxR.png)
![Second Preset Find Bar](https://i.imgur.com/uIrjy21.png)

I don't know if there's an easy solution. My ideas are either:
1. Adjust the preset colors to fix the issue.
2. Improve the WebExtension colors API to allow coloring the Find bar separately. At that point, we could either add new color fields for the find bar, or we could mix the toolbar and background colors based on transparency to get a Find bar background color, calculating what the EFFECTIVE toolbar color is (what the user would see the color as, assuming no background): `tinycolor.mix(toolbarColor, bgColor, toolbarAlpha)`

**Comment by codythecoder** [comment]
If the toolbar color is 100% transparent, and the toolbar icons and text, and search text are pure white, it becomes impossible to read the find bar. I really like your suggestion of coloring the find bar based on the bgcolor and toolbar color.

**Comment by johngruen** [comment]
@FayneAldan yeah these are not necessarily WCAG compliant...might be a good idea to make that the case, but they're designed to show off all the capabilities of FF color.
